doc/api-extensions.md

```diff
@@ -2329,3 +2329,7 @@ This introduces the configuration keys `cephfs.create_missing`, `cephfs.osd_pg_n
 
 This API extension provides the ability to use flags `--profile`, `--no-profile`, `--device`, and `--config`
 when moving an instance between projects and/or storage pools.
+
+## `existing_target_dir_mounted_by_disk_device`
+
+This API extension provides the ability to check if a target directory was created within the instance filesystem at mount time. If a host directory is mounted to an existing container directory (e.g., `/opt`), the target directory won't be removed upon unmounting. However, if LXD creates a new target directory during the mount, like `/new_dir`, it will be deleted when the device is unmounted.
```

Comment:
```suggestion
This API extension provides the ability to check if a target directory was created within the instance file system at mount time. If a host directory is mounted to an existing container directory (e.g., `/opt`), the target directory won't be removed upon unmounting. However, if LXD creates a target directory during the mount, like `/new_dir`, it will be deleted when the device is unmounted.
```